.github/workflows/ci.yaml

```diff
@@ -74,25 +74,13 @@ jobs:
           #   fails multiple times while others don't, its cache will likely get evicted,
           #   making it even slower to test and fix (uffff).
           #   When they fix this, we should remove ${{ github.run_id }} from the end of the key.
-          key: wasp-build-${{ runner.os }}-${{ matrix.ghc }}-${{ hashFiles('waspc/cabal.project.freeze') }}-${{ github.run_id }}
+          key: wasp-build-${{ runner.os }}-${{ matrix.ghc }}-${{ hashFiles('waspc/waspc.cabal') }}-${{ github.run_id }}
```

Comment:
Is this a good approach?

Reply:
Yeah true, I think we do want `cabal.project` as well in case we bump `index-state` but the actual dependencies remain the same. I will add that too thnx 👍🏻